site/data/docs/connect-button.mdx

```diff
@@ -59,6 +66,16 @@ The `ConnectButton` component exposes several props to customize its appearance
 
 ### Examples
 
+#### Label
+
+By default your Connect Button will say "Connect Wallet".
+
+For a custom label, pass your string to the `label` prop.
```

Comment:
could we use an assertive tone, for consistency with the rest of the docs?

something along the lines of: Use the `label` prop to set a custom `ConnectButton` text.

Reply:
for sure, sorry i know nothing about this kind of stuff